cli/doRun.go

```diff
@@ -415,12 +414,19 @@ func activateAndRunWorkflow(workflow models.WorkflowModel, bitriseConfig models.
 		if afterWorkflow.Title == "" {
 			afterWorkflow.Title = afterWorkflowName
 		}
-		buildRunResults = activateAndRunWorkflow(afterWorkflow, bitriseConfig, buildRunResults, environments)
+		buildRunResults = activateAndRunWorkflow(afterWorkflow, bitriseConfig, buildRunResults, environments, lastWorkflowTitle)
 	}
 
 	return buildRunResults
 }
 
+func lastWorkflowTitleInConfig(workflowToRun models.WorkflowModel, bitriseConfig models.BitriseDataModel) string {
```

Comment:
* please use/return the Workflow ID (which is unique) instead of the Title (which is not)
* make this recursive, multiple level of "after_run" is possible
* don't mix Title with ID (the `AfterRun` contains IDs, not titles)